mod_locations.asciidoc

```diff
@@ -6,43 +6,44 @@
 
 *Data owner: `CPO`*
 
-The Location objects live in the CPO back-end system. They describe the charging locations of that operator.
+The Location objects live in the CPO back-end system. They describe the charging locations of one operator (or of multiple operators in case of virtual CPOs).
```

Comment:
I think we will remove the term: Virtual CPO. The ideas it to describe `OCPI Platforms` and then each platform can provide services for different CPO's. So I don't think 1 CPO will report a Location that belongs to multiple CPO's. Then all the CPO's will have to report a Location with the same information GPS coordinates etc, but different EVSE's.

Reply:
I see how my addition can be read the way you construe it. As this is the initial description of the module, my intention was to cover all use-cases of the module. I didn't mean (and even considered) that data of multiple (virtual) CPOs could be mixed within the module or the protocol. To me this also doesn't make sense from a business perspective where I would find it very weird if company A is providing white-label services to company B and both companies A&B had a charging station at the very same location (or even if A provided white-label services to B&C and both B&C had a station at the same location).

I'll rephrase it to something more clear for now.